src/DynamoCoreWpf/Controls/DynamoTextBox.cs

```diff
@@ -419,16 +413,14 @@ private void OpenLibraryToolTipPopup(object sender, EventArgs e)
             // Moving tooltip back.
             HorizontalOffset--;
 
-            showTimer.Stop();
+            toolTipTimer.Stop();
```

Comment:
This `toolTipTimer.Stop();` is no longer needed since `LibraryToolTipTimer` ensures that the timer is stopped before raising the `TimerElapsed` event.